auctionapp/auction-app
# Complete WatchItems, PopularItems, and ListedItems components and add various other functionality/styling

#### What's this PR do?
- Completed WatchItems, PopularItems, and ListedItems components
- Added functionality to click on ItemEntry component to render Item component
- Added functionality to minimize the chat window
- Lots of styling
#### Where should the reviewer start?
- client/components/WatchItems.jsx, client/components/PopularItems.jsx, client/components/ListedItems.jsx
- client/components/ItemEntry.jsx, client/components/Item.jsx
- client/components/MessageBox.jsx
- public/styles.css
#### How should this be manually tested?
- Look at home page to check if watch, popular, and listed items are there
- Click on any item entry component and it should take you to the item page
- Play around with clicking the x and blue bar of the message window
